addOns/reports/CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
 
+### Added 
+- The Traditional JSON report, Traditional JSON Report with requests and responses, Traditional XML Report and Traditional XML Report with requests and responses now has "otherinfo" field per alert instance (Issue 7260).
+-Two test instances of the alert, thereby increasing the number of alert instances in test cases from one to two.
```

Comment:
This is not relevant for the user.

Reply:
The second bullet could just be dropped. Agreed this was the unit test change that you implemented but unit test stuff isn't relevant to end users.

Reply:
okk sure